packages/blockchain-wallet-v4-frontend/src/data/components/coinify/selectors.js

```diff
@@ -0,0 +1,12 @@
+import { compose, prop, path } from 'ramda'
+export const getCoinifyData = path(['components', 'coinify'])
+export const getCoinifyBusy = compose(prop('coinifyBusy'), getCoinifyData)
```

Comment:
This naming is not ideal. Is signup busy? quote? trade?

Reply:
it should represent any coinify interaction that can be loading/busy/success/notAsked. I don't think we need multiple pieces of state to handle this. more than one interaction should not be busy at the same time.